envoy/http/async_client.h

```diff
@@ -414,15 +437,15 @@ class AsyncClient {
       return *this;
     }
     RequestOptions& setParentSpan(Tracing::Span& parent_span) {
-      parent_span_ = &parent_span;
+      StreamOptions::setParentSpan(parent_span);
       return *this;
     }
     RequestOptions& setChildSpanName(const std::string& child_span_name) {
-      child_span_name_ = child_span_name;
+      StreamOptions::setChildSpanName(child_span_name);
       return *this;
     }
     RequestOptions& setSampled(absl::optional<bool> sampled) {
-      sampled_ = sampled;
+      StreamOptions::setSampled(sampled);
```

Comment:
Now, I think the `RequestOpions` could be an alias of `StreamOptions` because it has no any own field.